src/types/onyx/OnyxCommon.ts

```diff
@@ -25,10 +25,10 @@ type AvatarType = typeof CONST.ICON_TYPE_AVATAR | typeof CONST.ICON_TYPE_WORKSPA
 
 type Icon = {
     /** Avatar source to display */
-    source?: AvatarSource;
+    source: AvatarSource;
 
     /** Denotes whether it is an avatar or a workspace avatar */
-    type?: AvatarType;
+    type: AvatarType;
```

Comment:
Because originally they were not optional, and this was changed only recently in this PR: https://github.com/Expensify/App/pull/39637/files#diff-3b661ee1b05411e77221e1333e48cf2ced284539b5b835070c5ec397606e21a2 - which is like a week old, introduced some regressions and I believe it was a bad idea. There was no real need to make them optional.

IMO Icons that have no source and no type are kinda meaningless. If every field in `Icon` is optional then `{}` could be a correct icon and that can lead to bugs when there is no source and no accountID.

Yeah I understand that we could drop the `?? FallbackAvatar` which is ugly, but in general I feel unsafe with the Icon type that could have everything optional. I'd rather keep the explicit `??` and be always forced to put something inside icon.